src/commons/api/roas.rs

```diff
@@ -324,27 +340,27 @@ impl RoaDefinitionUpdates {
     }
 
     pub fn add(&mut self, add: RoaDefinition) {
-        self.added.insert(add);
+        self.added.push(add);
```

Comment:
Now that this is a Vec instead of a HashSet should this reject duplicates to maintain previous behaviour?